debezium-ddl-parser/src/test/resources/mysql/examples/bitrix_queries_cut.sql

```diff
@@ -1,2006 +0,0 @@
-#begin 1
```

Comment:
Why was the DDL parser SQL examples that are used to test the DDL parser with removed?